snippets/csharp/VS_Snippets_CLR_System/system.Console-REFORMAT/CS/reformat.cs

```diff
@@ -1,27 +1,33 @@
-﻿using System;
+using System;
 // This sample converts tab-delmited input and converts it to 
 // comma-delimited output.  Furthermore, it converts all boolean
 // input to numeric representations.
 // System.Console.Write
 // System.Console.WriteLine
 // System.Console.ReadLine
 // <Snippet1>
-public class FormatConverter {
-    public static void Main(string[] args) {
+public class FormatConverter
+{
+    public static void Main(string[] args)
+    {
         string lineInput;
-        while ((lineInput = Console.ReadLine()) != null) {
+        while ((lineInput = Console.ReadLine()) != null)
+        {
             string[] fields = lineInput.Split(new char[] {'\t'});
             bool isFirstField = true;
-            foreach(string item in fields) {
+            foreach (string item in fields)
+            {
                 if (isFirstField)
                     isFirstField = false;
                 else
                     Console.Write(',');
                 // If the field represents a boolean, replace with a numeric representation.
-                try {
+                try
```

Comment:
throwing an exception as a routine part of parsing a string that is expected to fail is not a good practice. This is .NET Framework 1.0 or 1.1 code, when no better option was available for this scenario. A better option here is to call Boolean.TryParse and execute the `Console.WriteLine(item)` if it returns false.